Dockerfile

```diff
@@ -10,38 +10,38 @@ COPY ./node-template/customSpec.json ./runtime/tools/spec-builder/customSpec.jso
 
 WORKDIR /usr/src/runtime
 
-RUN yarn install
-
-RUN yarn run build
-
-RUN mv wasm-code ./tools/spec-builder
+RUN yarn install && yarn run build && mv wasm-code ./tools/spec-builder
 
 WORKDIR /usr/src/runtime/tools/spec-builder
 
-RUN apt-get update -y 
-RUN apt-get install jq -y
-RUN yarn run asbuild && bash cr-custom-spec.sh
-RUN yarn build-spec -f customSpec.json
+RUN apt-get update -y && apt-get install jq -y
+RUN yarn run asbuild && bash cr-custom-spec.sh && yarn build-spec -f customSpec.json
 CMD ["/usr/bin/echo", "succesfully build as-runtime and generated chainspec file"]
 
 # Building node and running it with the customSpecRaw.json
 
 FROM rust:1.45 AS node-builder
 
-WORKDIR /usr/src/dummy
+WORKDIR /usr/src/node
 
 COPY ./node-template ./
 
 COPY --from=builder /usr/src/runtime/tools/spec-builder/customSpecRaw.json ./
 
 RUN bash ./scripts/init.sh
-RUN apt-get update -y 
-RUN apt-get -y install clang
-RUN apt-get -y install gcc
-RUN apt-get -y install cmake
+RUN apt-get update -y &&\
+    apt-get -y install clang &&\
+    apt-get -y install gcc &&\ 
+    apt-get -y install cmake
 
 RUN cargo build --release
 
-EXPOSE 9933/tcp
+CMD ["/usr/bin/echo", "succesfully build as-runtime and generated chainspec file"]
 
-ENTRYPOINT ["./target/release/node-template", "--chain=./customSpecRaw.json", "--rpc-port", "9933"]
\ No newline at end of file
+FROM ubuntu:latest
+WORKDIR /usr/src/app
+COPY --from=node-builder /usr/src/node/target/release/node-template ./node-template
+COPY --from=node-builder /usr/src/node/customSpecRaw.json ./customSpecRaw.json
+EXPOSE 9933
+EXPOSE 9944/tcp
```

Comment:
if you are opening the 9944 port for the web sockets, you need to pass it as hardcoded argument in the `entrypoint` command